src/math/Stack.cc

```diff
@@ -487,10 +536,18 @@ std::shared_ptr<image::MaskedImage<PixelT>> statisticsStack(image::MaskedImage<P
             std::vector<std::shared_ptr<image::MaskedImage<TYPE>>> & images, Property flags,                 \
             StatisticsControl const &sctrl, WeightVector const &wvector, image::MaskPixel,                   \
             image::MaskPixel);                                                                               \
+    template std::shared_ptr<image::MaskedImage<TYPE>> statisticsStack<TYPE>(                                \
+            std::vector<std::shared_ptr<image::MaskedImage<TYPE>>> & images, Property flags,                 \
+            StatisticsControl const &sctrl, WeightVector const &wvector, image::MaskPixel,                   \
+            std::vector<std::pair<image::MaskPixel, image::MaskPixel>> const &);                                                                               \
```

Comment:
Missing backslash in `#define`?

Reply:
Just bad whitespace, actually, but still worth fixing.